### DISCUSSION TITLE
while configuring the apiman ui using wildfly getting the error

### DISCUSSION THREAD

**Discussion by shachindrasingh** [apiman/apiman#2534] [General]
when i was using Open JDK 17  and run the wildfly i am getting the below error on logs.
java.lang.reflect.genericsignatureformaterror

when i check different blogs. i found the we need to user JDK 11 . when i use JDK 11 then it start working can you please help me in that

**Comment by msavy** [comment]
I don't understand what help you want?

**Reply by shachindrasingh** [reply]
Is current wildfly 23.0.3 support JDK 17 ? when i tried to run that i am getting the below error in wildfly logs.
java.lang.reflect.GenericSignatureFormatError: Signature Parse error: Expected Field Type Signature
	Remaining input: javax.transaction.Transactional$TxType"}}

**Reply by msavy** [reply]
WildFly 23 does not fully support Java 17. Just use Java 11.
